### ISSUE TITLE
[Feature Request] Option to assign priority override tags via game menu

### ISSUE THREAD

**Issue by daephx** [felixkmh/DuplicateHider#71]
Have the plugin maintain the default action of utilizing sources (and/or Libraries: [#issues/58](https://github.com/felixkmh/DuplicateHider/issues/58)) to determine which duplicate copy of a game has priority and should be displayed/launched first, but allow overrides for individual titles or series.

In my case I have some outliers in my library. I tend to prefer Steam but some game collections are mismatched between GOG, Epic or Steam.

One of these examples would be The Witcher series:
I own only The Witcher 2 on Steam while the rest of the games were originally purchases on GOG.
Thanks to the [GOG Backup copies for Witcher 1 & 2](https://www.gog.com/witcher/backup), I now own all Witcher titles on GOG, but due to the duplicate priority system, the Steam copy of Witcher 2 is displayed first.

I would like to override this to display the GOG copy first without setting GOG as the highest priority source library.

<details>
<summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/36192863/171266600-2dced0d0-173a-43dc-8e4e-6aacf0db6e09.png)

</details>

**Comment by felixkmh** [comment]
I think there are two ways to achive what you want. 
One way is to create a custom group that contains all copies, then in the plugin settings for DuplicateHider under the 'Costum Groups' tab, you can check the 'Keep Order' option for that group and reorder the games of that group as you like (via drag-and-drop). This order will then override the priorities. 
The other, easier way to do it is by adding the '[DH] High' tag to the game you want to prioritize.

**Comment by daephx** [comment]
@felixkmh Alright, Thanks for the info! I'll look into those options.
Ideally, it would be nice if whatever implementation works best is available as a context menu entry for the game/plugin.

**Comment by felixkmh** [comment]
Just to clarify, because I'm not a 100% sure this came across: these options are already available in DuplicateHider right now, those were not suggestions on what I might implement. I might add the option to assign the priority override tags via the game menu though.

**Comment by daephx** [comment]
No worries, It did translate. I just needed to take a second to re-read your response and mess around with the tags.

I managed to get the desired result after a minute, I wasn't sure how to access the metadata for the hidden game at first but it's good now! I do believe a menu entry would be better / more user friendly, so perhaps the issue could be renamed since the initial request is already implemented.

Otherwise, I'm good now. Thanks again!
